src/silx/gui/plot/test/testPlotWidget.py

```diff
@@ -1530,6 +1530,7 @@ def testBoundingRectArguments(self):
         with self.assertRaises(Exception):
             item.setBounds((-1000, 1000, 2000, -2000))
 
+    @pytest.mark.filterwarnings("ignore:Attempting to set identical low and high ylims makes transformation singular; automatically expanding.:UserWarning")
```

Comment:
Ignore warnings only during tests: Those 3 warnings are about erroneous conditions so thet are not ignored in silx code.